vaadin-side-nav-flow-parent/vaadin-side-nav-flow/src/test/java/com/vaadin/flow/component/sidenav/tests/SideNavItemTest.java

```diff
@@ -388,6 +389,88 @@ public void expandAndCollapse_isCollapsed() {
         Assert.assertFalse(sideNavItem.isExpanded());
     }
 
+    @Test
+    public void createWithNoPathAlias_pathAliasesEmpty() {
+        final SideNavItem item = new SideNavItem("Test");
+
+        Assert.assertEquals(0, item.getPathAliases().size());
+    }
+
+    @Test
+    public void addEmptyPathAlias_pathAliasAdded() {
+        final SideNavItem item = new SideNavItem("Test");
+        item.addPathAlias("");
+
+        Assert.assertEquals(1, item.getPathAliases().size());
+    }
+
+    @Test
+    public void addMultiplePathAliases_pathAliasesAdded() {
+        final SideNavItem item = new SideNavItem("Test");
+        item.addPathAlias("alias1", "alias2");
+
+        Assert.assertEquals(2, item.getPathAliases().size());
+    }
+
+    @Test
+    public void addPathAlias_removePathAlias_pathAliasesEmpty() {
+        final SideNavItem item = new SideNavItem("Test");
+        item.addPathAlias("alias");
+        item.removePathAlias("alias");
+
+        Assert.assertEquals(0, item.getPathAliases().size());
+    }
+
+    @Test
+    public void addMultiplePathAliases_clearPathAliases_pathAliasesEmpty() {
+        final SideNavItem item = new SideNavItem("Test");
+        item.addPathAlias("alias1", "alias2");
+        item.clearPathAliases();
+
+        Assert.assertEquals(0, item.getPathAliases().size());
+    }
+
+    @Test
+    public void setPathAsComponent_pathAliasesAdded() {
+        try (MockedStatic<ComponentUtil> mockComponentUtil = Mockito
+                .mockStatic(ComponentUtil.class)) {
+            mockComponentUtil.when(() -> ComponentUtil.getRouter(Mockito.any()))
+                    .thenReturn(router);
+
+            sideNavItem.setPath(TestRoute.class);
+
+            Assert.assertEquals(2, sideNavItem.getPathAliases().size());
```

Comment:
I wonder if we should check not only the size but also that the aliases are actually correct.

Reply:
Updated the tests.